src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -710,6 +710,29 @@ protected bool IsAlive
                 return !ElementIDLifecycleManager<int>.GetInstance().IsRevitDeleted(InternalElementId.IntegerValue);
             }
         }
+        /// <summary>
+        /// Finds the elements that are joined with the given element.
+        /// </summary>
+        /// <returns>all elements joined to the given element</returns>
+        public List<Element> GetJoinedElements()
+        {
+            if (this.InternalElement == null)
+                throw new NullReferenceException("Cannot get joined elements from a null Element");
+            if (Document == null)
+                throw new NullReferenceException("Cannot get joined elements from a null Element");
+
+            ElementId[] joinedElementIds = JoinGeometryUtils.GetJoinedElements(Document, this.InternalElement)
+                                                            .ToArray();
+            
+            List<Element> joinedElements = new List<Element>();
+            for (int i = 0; i < joinedElementIds.Length; i++)
+            {
+                joinedElements.Add(Document.GetElement(joinedElementIds[i]).ToDSType(true));
```

Comment:
also, if `GetJoinedElements` returns an enumerator, we might be able to do this at the same time instead of having a for loop run over it a second time.